src/Compilers/CSharp/Portable/Emitter/Model/PEAssemblyBuilder.cs

```diff
@@ -15,6 +18,7 @@ internal abstract class PEAssemblyBuilderBase : PEModuleBuilder, Cci.IAssemblyRe
     {
         private readonly SourceAssemblySymbol _sourceAssembly;
         private readonly ImmutableArray<NamedTypeSymbol> _additionalTypes;
+        private ImmutableDictionary<WellKnownMember, MethodSymbol> _embeddedAttributesConstructors;
```

Comment:
>private ImmutableDictionary<WellKnownMember, MethodSymbol> _embeddedAttributesConstructors; [](start = 8, length = 91)

This dictionary feels like an overkill. we just need to store two method symbols. Also, it is not worth making Embedded attribute well-known type just so we can store its constructor in this dictionary